src/main/java/UseCase/RegisterArtistInteractor.java

```diff
@@ -0,0 +1,41 @@
+package UseCase;
+
+import Database.userAccessInterface;
+import Database.userDsData;
+import Entities.User;
+import Entities.userFactory;
+import InputBoundary.RegisterArtistInputBoundary;
+import InputData.RegisterArtistInputData;
+import OutputBoundary.loginOutputBoundary;
+
+public class RegisterArtistInteractor implements RegisterArtistInputBoundary{
+    private final loginOutputBoundary registerPresenter;
+    private final userFactory userFactory;
+
```

Comment:
You haven't implemented this yet which is probably why you don't have it, but in case you want to know how to access the database -- 
- Add `private final userAccessInterface userList;` here
- In the initializer, `this.userList = Database.UserList.getInstance();`